samples/excel/85-preview-apis/dynamic-arrays.yaml

```diff
@@ -0,0 +1,185 @@
+          await Excel.run(async (context) => {
+            context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+            const sheet = context.workbook.worksheets.add("Sample");
+
+            const expensesTable = sheet.tables.add("A4:D4", true /*hasHeaders*/);
+            expensesTable.name = "ExpensesTable";
+
+            expensesTable.getHeaderRowRange().values = [["Date", "Merchant", "Category", "Amount"]];
+
+            expensesTable.rows.add(null /*add at the end*/, [
+              ["1/1/2020", "The Phone Company", "Communications", "$120"],
+              ["1/2/2020", "Northwind Electric Cars", "Transportation", "$142"],
+              ["1/5/2020", "Best For You Organics Company", "Groceries", "$27"],
+              ["1/10/2020", "Coho Vineyard", "Restaurant", "$33"],
+              ["1/11/2020", "Bellows College", "Education", "$350"],
+              ["1/15/2020", "Trey Research", "Other", "$135"],
+              ["1/15/2020", "Best For You Organics Company", "Groceries", "$97"]
+            ]);
+
+            sheet.getRange("A2:H2").values = [["Transactions", , , , , , "Category", "Groceries"]];
+            sheet.getRange("A2").style = "Heading1";
+            sheet.getRange("G2").style = "Heading2";
+            sheet.getRange("H2").format.fill.color = "#EEEE99";
+
+            sheet.getUsedRange().format.autofitColumns();
+            sheet.getUsedRange().format.autofitRows();
+
+            sheet.activate();
+            await context.sync();
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to apply formulas that return dynamic arrays and show how to get the relevant information
+            about range spilling from the used ranges.</p>
+        </section>
+
+        <section class="setup ms-font-m">
+          <h3>Set up</h3>
+          <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Create table</span>
+            </button>
+        </section>
+
+        <section class="samples ms-font-m">
+          <h3>Try it out</h3>
+          <button id="copy-table-headers" class="ms-Button">
+                  <span class="ms-Button-label">Copy table headers</span>
+              </button>
+          <p />
+          <button id="apply-filter-function" class="ms-Button">
+                <span class="ms-Button-label">Apply FILTER formula</span>
+            </button>
+          <p />
+          <p>Select a cell and press the following button to see any spill information for that cell related to dynamic
```

Comment:
I think we should clarify that the user needs to select a new, spilled cell for this to work. Maybe 'Select a spilled cell'?